README.md

```diff
@@ -5,7 +5,13 @@ Data protection is more and more essential which can protect and backup data aut
 However, the idea of that is hard to understand from scratch. With this chance, we would like to develop a very simple policy-driven data protection system to let everyone easily get familiar with the idea of that.
 
 So, we would like to implement a policy-driven data protection system with the following
+
 - Open source a very simple but clear policy-driven data protection system
 - A back-end to make sure we won't lose data if any computer/host is down
 - A back-end that users can upload and store objects regarding policies
 - A front-end page to let user upload and watch their data
+
+## Screenshots
+
+![Screenshot of frontend (dark)](/.github/assets/screenshot-dark.png#gh-dark-mode-only)
+![Screenshot of frontend (light)](/.github/assets/screenshot-light.png#gh-light-mode-only)
```

Comment:
I think we can add one more section for how to deploy frontend/backend through docker with a one-line command

Reply:
Done in [`cbffbde` (#2)](https://github.com/kaichiachen/Policy-driven-data-protection/pull/2/commits/cbffbdea01cd283319759d974e0a161b32ec56e4)